Plots/Contours/NCL_conwomap_3.py

```diff
@@ -0,0 +1,88 @@
+"""
+NCL_conwomap_3.py
+=================
+Concepts illustrated:
+  - Drawing a simple contour plot
+  - Generating dummy data using "random_normal"
+  - Masking mirrored contour data
+  - Drawing a perimeter around areas on a contour plot with missing data
+  - Turning off the bottom and right borders of a contour plot
+  - Using "getvalues" to retrieve resource values
```

Comment:
It looks like some of these comments apply to the original NCL script, but not the Python version. For example, I think "getvalues" is an NCL construct. Could you review the comments and remove the ones that aren't relevant?